src/openrct2-ui/input/KeyboardShortcuts.cpp

```diff
@@ -159,22 +159,22 @@ void KeyboardShortcuts::Set(int32_t key)
 
 int32_t KeyboardShortcuts::GetFromKey(int32_t key)
 {
-    for (int32_t i = 0; i < SHORTCUT_COUNT; i++)
+    for (size_t i = 0; i < GetShortcutsCount(); i++)
     {
         if (key == _keys[i])
         {
-            return i;
+            return static_cast<int32_t>(i);
         }
     }
-    return SHORTCUT_UNDEFINED;
+    return static_cast<int32_t>(Input::Shortcut::Undefined);
```

Comment:
this is not Input::Shortcut::Undefined but a different variable with the same value.